design/one-pager-pluggable-secret-stores.md

```diff
@@ -70,7 +69,7 @@ spec:
   type: External
```

Comment:
What do you think about renaming the type as I suggested [before](https://github.com/crossplane/crossplane-runtime/pull/379#discussion_r1111848656)? 

```suggestion
  type: Plugin
```